server/dev/config/cipher.aes

```diff
@@ -0,0 +1 @@
+e83d7343445b54ecdf4c5ca2afc31f47
```

Comment:
@ibnc yes, good catch. I quickly pushed this up yesterday so that @ketan could see what I was referring to in regards to my versioning question. I definitely didn't mean to include this file. I should've noted that this PR is a WIP